src/components/MovieBooking/MovieSeats.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Component should be more modular and reusable. Extract logic and UI separately for better testing. Make use of custom hooks for logical part.